### ISSUE TITLE
Viber support

### ISSUE THREAD

**Issue by mrisek** [infobip/infobip-api-java-client#19]
Hi. Does this library support sending Viber messages? If not, are there any plans to add Viber support to Infobip API client library?

**Comment by josipk-ib** [comment]
Hi Matija, apologies for late response. Viber is not yet supported by the library. We have just released new [Viber standalone API](https://www.infobip.com/docs/api/#channels/viber) this week and we plan to support for it in library soon. If you're in a hurry you can try to use API directly. Please keep an eye on this repository for upcoming features . Thanks for you interest in our services.

**Comment by mrisek** [comment]
Hi Josip, thank you for your advice. Looking forward to new features like Viber support.

**Comment by ib-fsrnec** [comment]
Hi @mrisek, just to let you know that you can expect Viber API in our next major release that's about to be released in few weeks. Thank you for your patience.

**Comment by ib-fsrnec** [comment]
Hi @mrisek, I'm happy to say that we released a new major version 4.0.0 yesterday.

We added the requested Viber support. Here is the full example describing how to send a simple text message:
```
        ApiClient infobipApiClient = ApiClient.forApiKey(ApiKey.from(givenApiKey))
                .withBaseUrl(givenBaseUrl))
                .build();

        ViberApi viberApi = new ViberApi(infobipApiClient);

        ViberTextMessage viberTextMessage = new ViberTextMessage()
                .content(new ViberTextContent().text("Hello World!"))
                .from(givenFrom)
                .to(givenDestination);

        ViberBulkTextMessage viberBulkTextMessage = new ViberBulkTextMessage()
                .addMessagesItem(viberTextMessage);

        ViberBulkMessageInfo messageInfo = viberApi
                .sendViberTextMessage(viberBulkTextMessage)
                .execute();
```

For additional details, please check the official API [documentation|https://www.infobip.com/docs/api/channels/viber].

I'm going to close the issue. Feel free to reopen it if you find out additional issues.

Best regards,
Filip
